app/scheduling/directives/termCalendar.js

```diff
@@ -253,11 +323,24 @@ schedulingApp.directive("termCalendar", this.termCalendar = function ($rootScope
 				return calendarActivities;
 			};
 
-			var styleCalendarEvents = function (calendarActivities, backgroundColor, borderColor, textColor) {
+			var styleCalendarEvents = function (calendarActivities, backgroundColor, borderColor, textColor, tagColor) {
```

Comment:
Add a comment indicating what this function is doing, e.g. sets text color, border Color, and background color based on UI selection state, etc.